mesosphere/dcos-docs-site
# D2IQ-67167: bump bootprint, bootprint-openapi deps

## Jira Ticket
https://jira.d2iq.com/browse/D2IQ-67167

## Description of changes being made
Updating bootprint dependencies, more details in JIRA.

## How to test:
- do `make build-api` **before** checking out this branch
- rename `build-ngindox` and `build-swagger` folders to save them for later comparison
- checkout this branch, `npm install` and `make build-api` again
- `diff -r build-ngindox build-ngindox_old` -> I got no changes
- `diff -r build-swagger build-swagger_old` -> seems like some css things have changed.
- `npm run dev` and **smoke test pages displaying DC/OS swagger stuff, make sure everything looks okay**
